src/python/espressomd/observables.py

```diff
@@ -14,15 +14,19 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import numpy as np
 from .script_interface import ScriptInterfaceHelper, script_interface_register
 
 
 @script_interface_register
 class Observable(ScriptInterfaceHelper):
     _so_name = "Observables::Observable"
-    _so_bind_methods = ("calculate", "n_values")
+    _so_bind_methods = ("calculate_flat_array", "shape")
```

Comment:
I don't like `calculate_flat_array`, what was wrong with `calculate`? `calculate_flat_array` suggests that it can also calculate something else, which is not the case.

Reply:
Should I rename it `_calculate`? We need a different name to avoid a name collision.

Reply:
With that would it collide?

Reply:
It wouldn't. I just tried it, but for some reason you can't have a method starting with an underscore in the `_so_bind_methods` attribute of a `script_interface_register`.

Reply:
nvm, forgot to update the two other scriptinterface files